velox/type/Date.cpp

```diff
@@ -27,15 +27,16 @@ std::string Date::toString() const {
   // Find the number of seconds for the days_;
   // Casting 86400 to int64 to handle overflows gracefully.
   int64_t daySeconds = days_ * (int64_t)(86400);
```

Comment:
As the comment in TEST(TypeTest, dateToString) says, it doesn't crash but returns garbage.
``` C++
  // Trying a very large -integer for boundary checks. Such values are tested in
  // ExpressionFuzzer.
  // Since we use int64 for the intermediate conversion of days to ms,
  // the large -ve value remains valid. However, gmtime uses int32
  // for the number of years, so the eventual results might look like garbage.
  // However, they are consistent with presto java so keeping the same
  // implementation.
  Date dateOverflow(-1855961014);
  EXPECT_EQ(dateOverflow.toString(), "-5079479-05-03");